### PR TITLE
Add pane increase/decrease size commands

### PR DESCRIPTION
**Pull request by kevinsawicki**
Adds two new pane commands, `pane:increase-size` and `pane:decrease-size` with some default keybindings on all platforms.

Continuation of #7354

Closes https://github.com/atom/atom/issues/7332

### CODE REVIEW

**Comment by kevinsawicki** [comment]
@izuzak :eyes: any concerns about these keybindings? They seem pretty intuitive to me.

**Comment by amfarrell** [comment]
As a user new to atom (coming from vim) I forked it to try out @liuxiong332's commits and they were pretty darn intuitive even though they don't match vim's :+1:

File: keymaps/linux.cson [hunk 1]
```diff
@@ -143,6 +143,10 @@
   'ctrl-k ctrl-8': 'editor:fold-at-indent-level-8'
   'ctrl-k ctrl-9': 'editor:fold-at-indent-level-9'
 
+'atom-workspace atom-pane':
+  'ctrl-alt-=': 'pane:enlarge'
+  'ctrl-alt--': 'pane:shrink'
```
Comment (izuzak) [inline keymaps/linux.cson]: Should these be `pane:increase-size` and `pane:decrease-size`, same as in the OSX keybindings?

File: keymaps/win32.cson [hunk 2]
```diff
@@ -147,6 +147,10 @@
   'ctrl-k ctrl-8': 'editor:fold-at-indent-level-8'
   'ctrl-k ctrl-9': 'editor:fold-at-indent-level-9'
 
+'atom-workspace atom-pane':
+  'ctrl-alt-=': 'pane:enlarge'
+  'ctrl-alt--': 'pane:shrink'
```
Comment (izuzak) [inline keymaps/win32.cson]: Should these be `pane:increase-size` and `pane:decrease-size`, same as in the OSX keybindings?

**Comment by izuzak** [comment]
> @izuzak  any concerns about these keybindings? They seem pretty intuitive to me.

Yeah, they seem intuitive to me too :+1: 

Also, just tried searching if any other packages use these keybindings and found some, and they all use those keybindings for resizing panes:

https://github.com/search?utf8=%E2%9C%93&q=%22ctrl-alt--%22+extension%3Acson&type=Code&ref=searchresults

File: keymaps/win32.cson (hunk 2 above)
Comment (kevinsawicki) [inline keymaps/win32.cson]: Whoops, absolutely, thanks for catching that :green_heart:

**Comment by kevinsawicki** [comment]
Thanks for this @liuxiong332 

And thanks @izuzak and @amfarrell for the feedback.

This will be included in the next Atom release, 1.0.8

**Comment by Glutexo** [comment]
On Windows with Czech keyboard, because of this binding it is effectively impossible to write an asterisk `*`. It is written as `AltGr`+`-` or `Ctrl`+`Alt`+`-`. `-` is located in the lower-right corner next to the Right Shift key.

**Pull request merged**